data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '02-03-2020', 0, TRUE, 10.23);
```

Comment:
- [ ] Kindy ensure you name your values correctly,this will be important to produce the correct query result.

Reply:
Done!
Thanks!